### ISSUE TITLE
Repeated sentence "Don't worry the tooling is open source and 100% free to use." in the same chapter

### ISSUE THREAD

**Issue by apupier** [camelinaction/camelinaction2#36]
In this chapter https://github.com/camelinaction/camelinaction2/tree/master/chapter19/eclipse-camel-editor
"Don't worry the tooling is open source and 100% free to use." is repeated 2 times. Maybe keeping it just one time is enough?

**Comment by davsclaus** [comment]
Yeah lets remove one of them. And maybe rephrase it as

    The tooling is open source and free to use

You are welcome to provide a PR
